includes/wikia/services/WikiDetailsService.class.php

```diff
@@ -262,6 +263,26 @@ protected function cacheWikiData( $wikiInfo, $method = null ) {
 	}
 
 	/**
+	 * @param int $id
+	 * @return mixed
+	 */
+	private function getDiscussionStats( $id ) {
+		global $wgConsulServiceTag, $wgConsulUrl, $wgEnableDiscussions;
+
+		if ( $wgEnableDiscussions ) {
+			$consulUrl = ( new Wikia\Service\Gateway\ConsulUrlProvider( $wgConsulUrl, $wgConsulServiceTag ))->getUrl( 'discussion' );
+			$response = Http::get( "http://$consulUrl/$id/forums/$id", 'default', array( 'noProxy' => true ));
+			if ( $response != false ) {
+				$decodedResponse = json_decode( $response, true );
+				if ( isset($decodedResponse) && json_last_error() === JSON_ERROR_NONE ) {
+					return $decodedResponse[ 'threadCount' ] ? $decodedResponse[ 'threadCount' ] : null;
```

Comment:
I'm not sure this condition is valid.
When ` $decodedResponse[ 'threadCount' ]` is `0` then we return `null`?